### PR TITLE
CSS for Product Information

### PR DESCRIPTION
**Pull request by claire123c**
- Added basic CSS for product information sections: Info, Freeform and Stars

Edge Cases/Bug Fixes:
- Fixed Image Gallery CSS

Misc.:
- Added test for star function
- Moved star function to separate folder

### CODE REVIEW

File: client/src/components/overview/Info/Info-helper.jsx/star-helper.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+const averageRating = (reviewResults, changeState) => {
+  let ratings = 0;
+  let totalRatings = 0;
+
+  if (reviewResults && reviewResults.length !== 0) {
+    for (let i = 0; i < reviewResults.length; i += 1) {
+      if (reviewResults[i].rating !== undefined) {
+        ratings += reviewResults[i].rating;
```
Comment (hollinwakefield) [inline client/src/components/overview/Info/Info-helper.jsx/star-helper.jsx:8]: Reminder that the reviewResults array is not currently taking into account all ratings

**Review by hollinwakefield — APPROVED** [review]
